client/src/components/Profile/EmploymentHistory/EmploymentHistoryDisplayItem.tsx

```diff
@@ -0,0 +1,42 @@
+import { Typography, Space } from 'antd';
+import { EmploymentRecordFormItem } from '../EmploymentCard';
+import { Dayjs } from 'dayjs';
+
+const workingPeriodFormat = 'MMM YYYY';
+const { Title, Text } = Typography;
+
+const EmploymentHistoryDisplayItem = ({
+  employmentRecord,
+}: {
+  employmentRecord: Omit<EmploymentRecordFormItem, 'officeLocation'>;
+}) => {
+  const { title, dateFrom, dateTo, toPresent, companyName } = employmentRecord;
+
+  if (!title && !companyName) {
+    return <Title level={5}>(Empty)</Title>;
+  }
+
+  const getWorkingPeriod = (dateFrom: Dayjs, dateTo: Dayjs): string => {
```

Comment:
plz move out the functions that are not related to rendering out of component body in all places

Reply:
Ok, thanks